src/components/common/list/list-item.component.tsx

```diff
@@ -1,7 +1,8 @@
 import classnames from 'classnames';
+import React from 'react';
 import styled from "styled-components";
 
-export default function ListItem({ listItem, activeKey }) {
+ const ListItem: React.FC<any> = ({ listItem, activeKey }) => {
```

Comment:
can we not use `export default function ListItem({ listItem, activeKey }): React.FC<any>` ?

Reply:
I think I misunderstood this. So basically, ListItem would be of type `React.FC<any>`, not it's return value. Marking this as resolved